README.md

```diff
@@ -1,3 +1,32 @@
+# Face Detector Service
+
+This service detect faces in images provided by the cameras.
+
+## Streams
+
+| Name | Input (topic/message) | Output (topic/message) | Description
+| --- |--- | --- | --- |
+|Face.Detection | **CameraGateway.\d+.Frame** [Image](https://github.com/labviros/is-msgs/blob/modern-cmake/docs/README.md#is.vision.Image) | **SkeletonsDetector.\d+.Detection** [Image](https://github.com/labviros/is-msgs/blob/modern-cmake/docs/README.md#is.vision.Image)|After detection, faces are drew on input image and published for visualization.
```

Comment:
Does the output is a Image?

Reply:
Yes, is a Image. The program detects the face, draw on the image and publish. Do you think it is necessary add another topic publishing only the bounding boxes?

Reply:
Yes!! I mean, you are publising an image on a topic named `SkeletonsDetector.\d+.Detection` (which is wrong!!! copy&paste hell hahahha). It suppose to have detections instead of an image. You can keep publishing an image with the detections, but publish it on a topic `*.Rendered` and the detections on a topic `*.Detections` containing this kind of message: [ObjectAnnotations](https://github.com/labviros/is-msgs/blob/master/docs/README.md#is.vision.ObjectAnnotations).

Please, let's close this PR and create another one to fix this! We are using PR to do a lot of unrelated stuff, that's a good pratice.